app/src/main/res/values/strings.xml

```diff
@@ -261,6 +261,10 @@ The info you enter is then directly added to the OpenStreetMap in your name, wit
 	<string name="quest_toiletsFee_title">"Do these toilets require a fee?"</string>
     <string name="quest_baby_changing_table_toilets_title">"Does this toilet have a baby changing table?"</string>
     <string name="quest_baby_changing_table_title">"Does \"%s\" have a baby changing table?"</string>
+    <string name="quest_way_light_title">"Is this piece of way lit?"</string>
+    <string name="quest_way_light_named_title">"Is \"%s\" way lit?"</string>
```

Comment:
It is very confusing to call any kind of streets "ways". Please split this up into ways and streets. I think it is not necessary to split it up in two quests but at least the title should be different depending on the type of way.

Also, I'd use "section" instead of "piece". At least I used this wording for another quest.